flink-sliding-feature-view-benchmark/README.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+This folder contains benchmarks to test the performance of the transformations that run
+with the Flink processor.
+
+
+# Prerequisites
+
+Prerequisites for running this example:
+- Unix-like operating system (e.g. Linux, Mac OS X)
+- Python 3.7
+
+# Step-By-Step Instructions
+
+Please execute the following commands under the `flink-benchmark` folder to run the 
+benchmarks.
+
+1. Install Feathub pip package.
+
+   ```bash
+   $ python -m pip install --upgrade feathub-nightly
+   ```
+
+2. Start the Flink cluster.
+
+   ```bash
+   $ docker-compose up -d
+   ```
+
+   After the Flink cluster has started, you should be able to navigate to the
+   web UI at [localhost:8081](http://localhost:8081) to view the Flink dashboard.
+
+3. Run the benchmark to test the performance of the transformations, e.g.
+
+   ```bash
+   # The script run the benchmark with total of 10 millions records as input
+   $ python main.py
+   
+   # You can specify the number of records, e.g. 1000
+   $ python main.py --records-num 1000
+   ```
+
```

Comment:
Would it be useful and consistent with other examples' README to provide `Checkout the outputs` section? Typically there should be some results that user can check after running this example.